src/plugins/cosmos/components/modals/Staking/views/Overview.tsx

```diff
@@ -100,50 +99,52 @@ export const Overview: React.FC<StakedProps> = ({
               apr={bnOrZero(validatorInfo?.apr)}
             />
           </Skeleton>
-          <Skeleton isLoaded={isLoaded} width='100%' mb='40px' justifyContent='space-between'>
-            <Box width='100%'>
-              <Text translation={'defi.rewards'} mb='12px' color='gray.500' />
-              <AssetClaimCard
-                assetSymbol={asset.symbol}
-                assetIcon={asset.icon}
-                cryptoRewardsAmount={bnOrZero(rewardsAmount)
-                  .div(`1e+${asset.precision}`)
-                  .decimalPlaces(asset.precision)}
-                fiatRate={bnOrZero(marketData.price)}
-                renderButton={() => (
-                  <ClaimButton
-                    assetId={assetId}
-                    validatorAddress={validatorAddress}
-                    // We're getting fractions of uatom as rewards, but at protocol-level, it is actually impossible to claim these
-                    // Any amount that's less than 1 uatom effectively means no rewards
-                    isDisabled={bnOrZero(rewardsAmount).lt(1)}
+        </Flex>
+      </ModalBody>
+      <ModalFooter>
```

Comment:
Why would we want to add all of this content inside the `ModalFooter`? It's not really some footer contents but more a part of the body content

Reply:
Yah this is totally correct :-P -- I'll fix it.

I like putting CTAs in the footer, and these are technically CTAs but its not really a big deal either way to me. I just like separating the content in some way could just as easily been another `ModalBody`